src/main/webapp/tech-survey.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Welcome First-Name</h1>
+    <h2>Tech Survey</h2>
+    <p>Rate your comfort with the following technologies:</p>
+    <form action="/action_page.php">
```

Comment:
This is ok for now, if there's time it would be nice to embed a Google Form, and use its API to get the results.

Reply:
Hmmm, the thing with google forms is that we cant style it so that its easier to select options/read

Reply:
That's true, it has limited styling.